modules/ROOT/pages/maintenance/commands/commands.adoc

```diff
@@ -64,6 +64,10 @@ Infinite Scale provides a xref:maintenance/commands/trash.adoc[CLI command] with
 
 Compared to offline commands that can be issued when Infinite Scale is shut down, online commands must be issued when Infinite Scale and the respective service(es) are running.
 
+=== Cleanup Orphaned Shares
+
+When a shared space or directory got deleted, use the following xref:maintenance/commands/shares-cleanup.adoc[CLI command] to cleanup those share orphans. This cant be done automatically at the moment.
```

Comment:
```suggestion
When a shared space or directory got deleted, use the following xref:maintenance/commands/shares-cleanup.adoc[CLI command] to cleanup those share orphans. This can't be done automatically at the moment.
```